soccer/src/soccer/strategy/agent/position/penalty_player.cpp

```diff
@@ -18,19 +18,30 @@ PenaltyPlayer::State PenaltyPlayer::update_state() {
             // if penalty playing and restart penalty in playstate we switch to shooting
             if (current_play_state_.is_ready() &&
                 (current_play_state_.is_penalty() || current_play_state_.is_kickoff())) {
-                return SHOOTING_START;
+                return SMALL_KICK;
             }
             break;
         }
-        case SHOOTING_START: {
+        case SMALL_KICK: {
+            if (distance_from_enemy_goal() < 3.5) {
```

Comment:
Magic number (3.5) - let's make this a constant, so we can change later if we want to! See how other files do this with variables that start with k.